examples/feature_demo/multi_select.py

```diff
@@ -92,7 +96,19 @@ def random_rotation():
     )
 
 
+initialized = False
+
+
 def animate():
+    global initialized
+
+    # Render with both materials, to initialize them.
```

Comment:
Would be awesome if there was some way to do thus without rendering a frame... Like some games have a initial loading screen where all future shaders and pipelines are precompiled.

Reply:
I made a quick start, but this quickly becomes bigger than fits the scope of this PR. I offloaded the idea into an issue: https://github.com/pygfx/pygfx/issues/941